community-contributions/scriptnodeSetup/README.md

```diff
@@ -0,0 +1,14 @@
+## Installation with script
+
+Login as root, then do:
+```
+wget https://raw.githubusercontent.com/IeSua-lab/community-repo/scriptnodeSetup/community-contributions/scriptnodeSetup/joystream-node-install.sh
```

Comment:
Could you remove this link, or change to this repo instead? I'm a little reluctant to have people to run a script that may be changed :)

Reply:
Removed the link. Changed to "Download joystream-node-install.sh"
Is that okay?